frontend/app/components/crud-list-item/component.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember'
+
+export default Ember.Component.extend({
+  tagName: '',
+  maskPassword: true,
+  actions: {
+    edit() {
+      this.set('edit', true)
```

Comment:
For such simple cases it might be better to just use `(mut edit true)` in the template